include/systems/equation_systems.h

```diff
@@ -454,6 +454,24 @@ class EquationSystems : public ReferenceCountedObject<EquationSystems>,
    **/
   void allgather ();
 
+  /**
+   * Calls to reinit() will also do two-step coarsen-then-refine
+   **/
+  void enable_refine_in_reinit()
```

Comment:
Yeah; it might be useful to temporarily disable; with just an enable() method once you've enabled you're committed.